src/perform_step/low_order_rk_perform_step.jl

```diff
@@ -817,10 +817,10 @@ end
     g7 = u
     # Hairer II, page 22
     ϱu, ϱd = zero(eltype(k7))^2, zero(eltype(g7))^2
-    @. g6 = (g7 - g6)^2
-    ϱd = sum(g6)
-    @. tmp = (k7 - k6)^2
-    ϱu = sum(tmp)	
+    @. g6 = g7 - g6
+    ϱd = integrator.opts.internalnorm(g6,t)
+    @. tmp = k7 - k6
+    ϱu = integrator.opts.internalnorm(tmp,t)
     integrator.eigen_est = sqrt(ϱu/ϱd)*oneunit(t)
```

Comment:
I think you need to drop the sqrt here

Reply:
Also noticing that `oneunit(t)` is only used here and not in similar code for other algorithms. Any reason for that?